main.go

```diff
@@ -272,13 +290,14 @@ func main() {
 			log.Printf("len = %d", len(buf))
 			log.Printf("Received broadcasted message with hash : %s", hash)
 			
-			var mirrors []mirror
+			var mirrors, retrymirrors []mirror
 			var localMirrorAddresses mirrorList			
 			lock2.RLock()
 			localMirrorAddresses = make(mirrorList, len(mirrorAddresses))
 			copy(localMirrorAddresses, mirrorAddresses)
 			lock2.RUnlock()
 			
+			var retryMirroraddresses  []string
```

Comment:
If we follow camelCase in our go code, then this should be `retryMirrorAddresses`